recipes/mfpymake/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "mfpymake" %}
+{% set version = "1.2.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/mfpymake-{{ version }}.tar.gz
+  sha256: 116c58158f6c0051fe27e68751d763f4f1906919cee3414a5d28b350ac8b29d5
+
+build:
+  entry_points:
+    - mfpymake = pymake.cmds.mfpymakecli:main
+    - make-program = pymake.cmds.build:main
+    - make-code-json = pymake.cmds.createjson:main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - setuptools >=45
+    - wheel
+    - pip
+  run:
+    - python >=3.8
+    - numpy
+    - requests
+    - networkx >=2.6.3
+    - meson
+    - ninja
+    - pydotplus
+
+test:
+  imports:
+    - pymake
+  commands:
+    - mfpymake --help
+    - make-program --help
+    - make-code-json --help
+
+about:
+  home: https://github.com/modflowpy/pymake
+  summary: pymake is a Python package to compile MODFLOW-based models.
+  license: Unlicense
```

Looks like it actually should be:
```suggestion
  license: CC0-1.0
```